docs/tables/crowdstrike_detection.md

```diff
@@ -0,0 +1,93 @@
+# Table: crowdstrike_detection
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  detection_id,
+  created_timestamp
+from
+  crowdstrike_detection
+```
+
+### List detections from the last 3 months
+
+```sql
+select
+  detection_id,
+  created_timestamp
+from
+  crowdstrike_detection
+where
+  created_timestamp > current_date - interval '3 months';
+```
+
+### List high severity detections
+
+```sql
+select
+  detection_id,
+  created_timestamp,
+  device ->> 'device_id' as device_id
+from
+  crowdstrike_detection
+where
+  max_severity = 50;
```

Comment:
Is 50 the highest severity? Does it range from 0 - 50?

Reply:
I couldn't find any documentation on the highest possible value. Put `50` in as a rought estimate